tests/conflicts.rs

```diff
@@ -5,14 +5,14 @@ use clap::{App, Arg, ArgGroup, ErrorKind};
 static CONFLICT_ERR: &str = "error: The argument '-F' cannot be used with '--flag'
 
 USAGE:
-    clap-test <positional> <positional2> --flag --long-option-2 <option2>
+    clap-test <positional> --long-option-2 <option2> -F
 
 For more information try --help";
 
-static CONFLICT_ERR_REV: &str = "error: The argument '--flag' cannot be used with '-F'
+static CONFLICT_ERR_REV: &str = "error: The argument '-F' cannot be used with '--flag'
```

Comment:
I think the original message is correct here.

Reply:
That's right

Reply:
I don't know how these error messages should look like

Reply:
`-F` and `--flag` are swapped. They shouldn't be.